api-docs/api-reference/methods/users/post-forgot-pwd-v2.0.rst

```diff
@@ -0,0 +1,107 @@
+.. _post-forgot-pwd-v2.0:
+
+Forgot password
+~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/RAX-AUTH/forgot-pwd
+
+Use the Forgot password operation to get a password reset token delivered to
+the email address associated with your account.
+
+There are currently 2 flows through reset password.
+
+    - Via Astra/REACH (portal=astra_prod): When this portal value is used the
+      specified user will receive an email that contains a link. The link takes
```

Comment:
For the external customers:  Astra/REACH would be the Cloud Control Panel, right?   

Suggested revision:

You can use the reset password operation to reset the password through an interface, or by using the API. 
- If submit the request with  ``portal=astra_prod`` attribute, the specified user receives an e-mail that contains a link to reset the password through the user interface.
- If you submit the request without the portal attribute, the user receives an e-mail that contains a reset password token with instructions on how to use it in the :ref: `Reset password <post-reset-pwd-v2.0>` request.